extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/GrpcServerRecorder.java

```diff
@@ -216,10 +218,14 @@ private void prodStart(GrpcContainer grpcContainer, Vertx vertx, GrpcServerConfi
 
     private void postStartup(GrpcServerConfiguration configuration, GrpcBuilderProvider<?> provider, boolean test) {
         initHealthStorage();
-        LOGGER.infof("gRPC Server started on %s:%d [%s]",
-                configuration.host,
-                test ? configuration.testPort : configuration.port,
-                provider != null ? provider.serverInfo() : "SSL enabled: " + !configuration.plainText);
+        int port = test ? configuration.testPort : configuration.port;
+        String msg = "Started ";
+        if (provider != null)
+            msg += provider.serverInfo(configuration.host, port, configuration);
+        else
+            msg += String.format("gRPC server on %s:%d [%s]",
+                    configuration.host, port, "SSL enabled: " + !configuration.plainText);
```

Comment:
Unrelated to that PR but as you change that line, can you please use TLS instead of SSL? SSL died in 2015...